content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -520,71 +524,363 @@ Cancellation process
+information.
+
+First, select :guilabel:`Individual` as the :guilabel:`Company Type`. Then, add the
+:guilabel:`Operator License` in the :guilabel:`Accounting` tab of the contact form.
+
+For the customer address, make sure the following fields are complete:
+
+- :guilabel:`District`
+- :guilabel:`Tax ID` (:guilabel:`DNI`/:guilabel:`RUC`)
+- :guilabel:`Tax ID Number`
+
+.. image:: peru/operator-configuration.png
+   :align: center
+   :alt: Individual type operator configurations in the Contact form.
+
+Carrier
+*******
+
+The *carrier* is used when the delivery guide is through *public* transport.
+
+To create a new carrier, navigate to :menuselection:`Contacts --> Create` and fill out the contact
+information.
+
+First, select :guilabel:`Company` as the :guilabel:`Company Type`. Then, add the :guilabel:`MTC
+Registration Number`, :guilabel:`Authorization Issuing Entity`, and the :guilabel:`Authorization
+Number`.
+
+For the company address, make sure the following fields are complete:
+
+- :guilabel:`District`
+- :guilabel:`Tax ID` (:guilabel:`DNI`/:guilabel:`RUC`)
+- :guilabel:`Tax ID Number`
+
+.. image:: peru/company-operator-configuration.png
+   :align: center
+   :alt:  Company type operator configurations in the Contact form.
+
+Vehicles
+********
+
+To configure the available vehicles, navigate to :menuselection:`Inventory --> Configuration -->
+Vehicles` and fill in the vehicle form with the information needed for the vehicle:
+
+- :guilabel:`Vehicle Name`
+- :guilabel:`License Plate`
+- :guilabel:`Is M1 or L?`
+- :guilabel:`Special Authorization Issuing Entity`
+- :guilabel:`Authorization Number`
+- :guilabel:`Default Operator`
+- :guilabel:`Company`
+
+.. important::
+   It is important to check the :guilabel:`Is M1 or L?` checkbox if the vehicle has less than four
+   wheels or less than eight seats.
+
+.. image:: peru/vehicle-not-m1-or-l-pe.png
+   :align: center
+   :alt: Vehicle not selected as an M1 or L type with extra fields shown.
+
+Products
+********
+
+To configure the available products, navigate to :menuselection:`Inventory --> Products` and open
+the product to be configured.
+
+Make sure that the applicable information in the product form is fully configured. The
+:guilabel:`Partida Arancelaria` (Tariff Item) field needs to be completed.
+
+Generating a GRE
+~~~~~~~~~~~~~~~~
+
+Once the delivery from inventory is created during the sales workflow, make sure you complete the
+|GRE| fields on the top-right section of the transfer form for the fields:
+
+- :guilabel:`Transport Type`
+- :guilabel:`Reason for Transfer`
+- :guilabel:`Departure start date`
+
+It is also required to complete the :guilabel:`Vehicle` and :guilabel:`Operator` fields under the
+:guilabel:`Guia de Remision PE` tab.
+
+The Delivery transfer has to be marked as *Done* for the :guilabel:`Generar Guia de Remision` button
+to appear on the left menu of the transfer form.
+
```

Comment:
Hello @samueljlieber  is it possible we can add the image generate_GRE_transferview.png after this section? If it was taken out due to a lot of screenshots it is better to have this one instead of the gre-delivery-slip.png, Let me know!